### PR TITLE
fix(oopif): store child frame id between frameDetached and attachedToTarget

### PR DESCRIPTION
**Pull request by yury-s**
#4393

### CODE REVIEW

File: src/server/chromium/crPage.ts [hunk 1]
```diff
@@ -584,7 +607,15 @@ class FrameSession {
       // Frame id equals target id.
       const targetId = event.targetInfo.targetId;
       const frame = this._page._frameManager.frame(targetId)!;
```
Comment (dgozman) [inline src/server/chromium/crPage.ts:609]: nit: remove "!"

File: src/server/chromium/crPage.ts [hunk 2]
```diff
@@ -542,12 +544,33 @@ class FrameSession {
...
+        continue;
+      if ((frame as any)[swappingOutChildFrames].delete(targetId))
+        return frame._id;
+    }
+    return null;
+  }
+
+  private _frameMaybeSwappingOut(frameId: string) {
+    const frame = this._page._frameManager.frame(frameId);
+    if (!frame) return;
```
Comment (dgozman) [inline src/server/chromium/crPage.ts:559]: style: return on the next line. I am surprised that eslint does not complain...

**Review by dgozman — APPROVED** [review]

File: src/server/chromium/crPage.ts (hunk 2 above)
Comment (yury-s) [reply src/server/chromium/crPage.ts:559]: yeah, I relied on it

File: src/server/chromium/crPage.ts (hunk 1 above)
Comment (yury-s) [reply src/server/chromium/crPage.ts:609]: done

**Pull request merged**